selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/SeleniumWebDriverHelper.java

```diff
@@ -148,6 +151,52 @@ public WebElement waitVisibility(WebElement webElement) {
     return waitVisibility(webElement, DEFAULT_TIMEOUT);
   }
 
+  /**
+   * Waits until {@link WebElement} with provided locator {@link By} be invisible
+   *
+   * @param elementLocator
+   * @param timeout waiting time in seconds
+   * @return {@code true} - if not visible,
+   *     <p>throws {@link org.openqa.selenium.TimeoutException} - if visible.
+   */
+  public boolean waitInvisibility(By elementLocator, int timeout) {
+    return webDriverWaitFactory.get(timeout).until(invisibilityOfElementLocated(elementLocator));
+  }
+
+  /**
+   * Waits until {@link WebElement} with provided locator {@link By} be invisible
+   *
+   * @param elementLocator
+   * @return {@code true} - if not visible,
+   *     <p>throws {@link org.openqa.selenium.TimeoutException} - if visible.
+   */
+  public boolean waitInvisibility(By elementLocator) {
+    return waitInvisibility(elementLocator, DEFAULT_TIMEOUT);
+  }
+
+  /**
+   * Waits until provided {@link WebElement} be invisible
+   *
+   * @param webElement
+   * @param timeout waiting time in seconds
+   * @return {@code true} - if not visible,
+   *     <p>throws {@link org.openqa.selenium.TimeoutException} - if visible.
+   */
+  public boolean waitInvisibility(WebElement webElement, int timeout) {
```

Comment:
returning `boolean` type is useless here because it can only return **true**, or throw timeout exception.

Reply:
In this case all usages of this method need check with Assert.assertTrue()

Reply:
What is the benefit from `Assert.assertTrue(true)`? It checks nothing.

Reply:
Assert.assertTrue(waitInvisibility)

Reply:
Actually, we can simply replace `Assert.assertTrue(waitInvisibility)` by `waitInvisibility` without regression because in our case `Assert.assertTrue(waitInvisibility)` <-> `Assert.assertTrue(true)`.